.github/run_test.sh

```diff
@@ -10,44 +10,59 @@ PODMAN_OPTS="--systemd=true --privileged"
 
 # FIXME: test_wireless has been removed because is not supported on CI
 # container EL7. We need to add this back for EL8 or greater.
-EXCLUDE_TESTS='
--e tests/tests_802_1x_updated_nm.yml
+EXCLUDE_TESTS_C7='
 -e tests/tests_auto_gateway_initscripts.yml
 -e tests/tests_bond_deprecated_initscripts.yml
+-e tests/tests_bond_initscripts.yml
+-e tests/tests_team_nm.yml
+-e tests/tests_unit.yml
+-e tests/tests_wireless_nm.yml
+'
+
+EXCLUDE_TESTS_C8S='
+-e tests/tests_auto_gateway_nm.yml
 -e tests/tests_bond_deprecated_nm.yml
+-e tests/tests_bond_deprecated_initscripts.yml
 -e tests/tests_bond_initscripts.yml
--e tests/tests_default_initscripts.yml
--e tests/tests_default_nm.yml
--e tests/tests_default.yml
+-e tests/tests_bond_nm.yml
+-e tests/tests_bond_options_nm.yml
+-e tests/tests_change_indication_on_repeat_run.yml
 -e tests/tests_ethernet_initscripts.yml
 -e tests/tests_ethernet_nm.yml
--e tests/tests_eth_pci_address_match_nm.yml
 -e tests/tests_ethtool_coalesce_initscripts.yml
+-e tests/tests_ethtool_coalesce_nm.yml
 -e tests/tests_ethtool_features_initscripts.yml
+-e tests/tests_ethtool_features_nm.yml
 -e tests/tests_ethtool_ring_initscripts.yml
+-e tests/tests_ethtool_ring_nm.yml
+-e tests/tests_eth_dns_support_nm.yml
+-e tests/tests_eth_pci_address_match_nm.yml
```

Comment:
What's the reason to exclude more tests?

Reply:
The reason is that I wrongly enabled a test, and such a test left the system in a very bad state which caused a series of failure like this https://github.com/linux-system-roles/network/pull/484#issuecomment-1073833334. (Which test it is, I will verify it in the VM)

Reply:
The reason is that I wrongly enabled a test, and such a test left the system in a very bad state which caused a series of failure like this #484 (comment). (Which test it is, I will verify it in the VM)